spec/forms/waste_exemptions_engine/applicant_name_form_spec.rb

```diff
@@ -10,27 +10,27 @@ module WasteExemptionsEngine
       subject(:validators) { form._validators }
 
       it "validates the first name using the PersonNameValidator class" do
-        expect(validators.keys).to include(:first_name)
-        expect(validators[:first_name].first.class)
+        expect(validators.keys).to include(:applicant_first_name)
+        expect(validators[:applicant_first_name].first.class)
           .to eq(WasteExemptionsEngine::PersonNameValidator)
       end
 
       it "validates the last name using the PersonNameValidator class" do
-        expect(validators.keys).to include(:last_name)
-        expect(validators[:first_name].first.class)
+        expect(validators.keys).to include(:applicant_last_name)
+        expect(validators[:applicant_first_name].first.class)
```

Comment:
```suggestion
        expect(validators[:applicant_last_name].first.class)
```

Old bug, but just spotted this!